Attendance/Plugins/ProjectTemplatePlugin/ProjectDescriptionHelpers/infoPlist/Project+InfoPlist.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Project+InfoPlist.swift
+//  Plugins
+//
+//  Created by Wonji Suh  on 3/22/25.
+//
+
+import Foundation
+import ProjectDescription
+
+public extension InfoPlist {
+  static let appInfoPlist: Self = .extendingDefault(
+    with: InfoPlistDictionary()
+      .setUIUserInterfaceStyle("Light")
+      .setCFBundleDevelopmentRegion("$(DEVELOPMENT_LANGUAGE)")
+      .setCFBundleExecutable("$(EXECUTABLE_NAME)")
+      .setCFBundleIdentifier("$(PRODUCT_BUNDLE_IDENTIFIER)")
+      .setCFBundleInfoDictionaryVersion("6.0")
+      .setCFBundleName("DDD 출석")
+      .setCFBundlePackageType("APPL")
+      .setCFBundleShortVersionString(.appVersion())
+      .setAppTransportSecurity()
+      .setCFBundleURLTypes()
+      .setAppUseExemptEncryption(value: false)
+      .setCFBundleVersion(.appBuildVersion())
+      .setLSRequiresIPhoneOS(true)
+      .setUIAppFonts(["PretendardVariable.ttf"])
+      .setUIApplicationSceneManifest([
+        "UIApplicationSupportsMultipleScenes": true,
+        "UISceneConfigurations": [
+          "UIWindowSceneSessionRoleApplication": [
+            [
+              "UISceneConfigurationName": "Default Configuration",
+              "UISceneDelegateClassName": "$(PRODUCT_MODULE_NAME).SceneDelegate"
+            ]
+          ]
+        ]
+      ])
```

Comment:
```suggestion
      .setUIApplicationSceneManifest([
        "UISceneConfigurations": [
          "UIWindowSceneSessionRoleApplication": [
            [
              "UISceneConfigurationName": "Default Configuration",
            ]
          ]
        ]
      ])
```

저희가 MutilpleScene이나 SceneDelegate 파일을 사용하지 않아서 이렇게 수정하는게 좋아보여요!
`SceneDelegate.swift` 파일이 주석상태로 남아있는데, 이것도 같이 삭제하시죠!